.travis.yml

```diff
@@ -52,7 +52,7 @@ before_install:
 
       diffstr=$(git diff $branch remotes/origin/master --name-only  -- '*.yml' --no-pager);
       changedfiles=($diffstr);
-      excludedList="vm_create_existingvnet_deployjavaapp.yml .travis.yml aks_create_scale.yml webapp.yml rest/sql-managed-instance.yml vm_create_image.yml";
+      excludedList="vm_create_existingvnet_deployjavaapp.yml .travis.yml aks_create_scale.yml webapp.yml rest/sql-managed-instance.yml vm_create_image.yml appgateway_create.yml";
```

Comment:
why appgate is excluded?

Reply:
because preview moduled don't have right version yet....

Reply:
preview_modules should keep sync with upstream, pls fix preview_module firstly